### PR TITLE
[bn] Localizing strings in bangla

### LINKED ISSUE #44993: [bn] We need to localize some content and make necessary adjustments before launching in Bengali.
<h1> What is this issue for?</h1> 

### Here, we are Localizing the Kubernetes website to make it available in Bengali.

<h1>How can I contribute?</h1>

All the information you need is covered  in this [video.](https://youtu.be/zlHNWVQwAj0)
For more information join us on [slack](https://kubernetes.slack.com/messages/kubernetes-docs-bn)

<h1> Checklist to translate a term</h1>

1. Check that no one else has claimed the page in the comments below.
2. Comment below the page that you would like to localize, focus on one page per PR, at a time.
   Example: `I  am working on content/bn/<xyz.md>`
4. Maintainers will add your username next to your selected page in the below checklist.
5. Fork this repo, make your localizations, and submit a pull request following the name convention [bn]  Localization of <file_name.md>.
  Example: `[bn] Localization of cloud_native_apps.md`
6. Target branch dev-1.24-bn.1 when creating the PR
7. tag/ping a maintainer to make it easier to track progress
8. Also, when you create a PR, please make sure to add these  line in the description.

   `Resolves : #44993 `

<h1> Minimum required content before go live in bn </h1>

```[tasklist]
### Concepts
- [x] `content/en/docs/_index.md`  @asem-hamid
- [x] `content/en/docs/concepts/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/architecture/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/containers/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/cluster-administration/_index.md` @sajjadrahman56
- [x] `content/en/docs/concepts/configuration/_index.md` @azaynul10
- [x] `content/en/docs/concepts/overview/_index.md` @sajibAdhi
- [x] `content/en/docs/concepts/overview/working-with-objects/_index.md` @cloudwithtanvir
- [x] `content/en/docs/concepts/policy/_index.md` @sajjadrahman56
- [x] `content/en/docs/concepts/scheduling-eviction/_index.md` @azaynul10
- [x] `content/en/docs/concepts/security/_index.md` @itsPronay
- [x] `content/en/docs/concepts/services-networking/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/storage/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/windows/_index.md` @sajjadrahman56
- [x] `content/en/docs/concepts/workloads/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/workloads/controllers/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/workloads/pods/_index.md` @sajjadrahman56
- [x] `content/en/docs/concepts/extend-kubernetes/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/extend-kubernetes/api-extension/_index.md` @asem-hamid
- [x] `content/en/docs/concepts/extend-kubernetes/compute-storage-net/_index.md` @asem-hamid
```

```[tasklist]
### Tasks
- [x] `included/optional-kubectl-configs-pwsh.md` @asem-hamid
- [x] `docs/tasks/tools/included/optional-kubectl-configs-fish.md` @asem-hamid
- [x] update `docs/tasks /_index.md` @asem-hamid
```

```[tasklist]
### Releases
- [x] `content/bn/releases/notes.md` @asem-hamid
- [x] `content/bn/releases/download.md` @asem-hamid
- [x] `content/bn/releases/version-skew-policy.md` @sajjadrahman56
- [x] `content/bn/releases/patch-releases.md` @itsPronay
- [x] `content/bn/releases/release.md` @radiumSodium
- [x] `content/bn/releases/release-managers.md` @itsPronay
```

```[tasklist]
### Releases -  Update
- [x] update `content/bn/releases/_index.md` @saifulshark
```

```[tasklist]
### includes
- [x] `default-storage-class-prereqs.md` @asem-hamid
- [x] `federation-deprecation-warning-note.md` @itsPronay
- [x] `index.md` @itsPronay
- [x] `task-tutorial-prereqs-node-upgrade.md`  @asem-hamid
- [x] `user-guide-content-moved.md` @asem-hamid
```

```[tasklist]
### Tutorials
- [x] `content/en/docs/tutorials/configuration/_index.md` @asem-hamid
- [x] `content/en/docs/tutorials/security/_index.md` @asem-hamid
- [x] `content/en/docs/tutorials/stateful-application/_index.md` @asem-hamid
- [x] `content/en/docs/tutorials/stateless-application/_index.md` @asem-hamid
- [x] `content/en/docs/tutorials/services/_index.md` @asem-hamid
```

```[tasklist]
### Setup
- [x] `content/en/docs/setup/production-environment/tools/_index.md` @sajibAdhi
- [x] `content/en/docs/setup/production-environment/tools/kubeadm/_index.md` @azaynul10
```

```[tasklist]
### Setup -  Update
- [x] update `content/bn/docs/setup/production-environment/_index.md` @sajibAdhi
- [x] update `content/bn/docs/setup/_index.md` @azaynul10
- [x] update `content/bn/docs/setup/production-environment/turnkey-solutions.md` @azaynul10
```

```[tasklist]
### More
- [x] `content/en/community/_index.html` @asem-hamid
- [x] `content/en/community/code-of-conduct.md` @asem-hamid
- [x] update `readme-bn.md` @asem-hamid
- [x] update `bn/docs/contribute /_index.md` @asem-hamid
- [x] update `bn/docs/home /_index.md` @asem-hamid
- [x] update `bn/docs/home /supported-doc-versions.md` @asem-hamid
- [x] `content/bn/_common-resources/index.md` @asem-hamid
- [x] `content/en/training /_index.html` @asem-hamid
- [x] `content/en/blog/_index.md` @azaynul10
- [x] `content/en/partners/_index.html` @itsPronay
- [x] `content/en/partners/_index.html` @asem-hamid (completed in this [PR](https://github.com/kubernetes/website/pull/46186))
- [x] update `content/bn/docs/reference/_index.md` @asem-hamid
- [x] `content/en/search.md` @sajjadrahman56
- [x] `data/i18n/bn/bn.toml` @asem-hamid
- [x] add files at `content/bn/case-studies` @Nafisa-Tabassum-Mim
- [x] add files at `content/bn/examples` @SAShameem
- [x] add `docs /images/` @asem-hamid
- [x] `Blog - 2023-12-18-read-write-once-pod-access-mode-ga.md` @asem-hamid
```

<h2> Glossary  </h2>

We have a total of 146 glossary terms to localize into Bangla. You can find them in this [issue](https://github.com/kubernetes/website/issues/45729)

### Related issue :
1. #45009 
2. #31677
3. #45729

### PR DESCRIPTION
**Pull request by asem-hamid**
Localizing strings in bangla 

fixes #44993

### CODE REVIEW

File: data/i18n/bn/bn.toml [hunk 1]
```diff
@@ -29,29 +53,32 @@ other = "Slack"
 other = "Stack Overflow"
```
Comment (sajjadrahman56) [inline data/i18n/bn/bn.toml:53]: স্ট্যাক ওভারফলো

File: data/i18n/bn/bn.toml [hunk 2]
```diff
@@ -60,16 +87,22 @@ other = "02 Jan 2006 15:04:05 MST"
...
 other = " ডকুমেন্টেশন আর সক্রিয়ভাবে রক্ষণাবেক্ষণ করা হয় না। আপনি বর্তমানে যে সংস্করণটি দেখছেন সেটি একটি স্ট্যাটিক স্ন্যাপশট। আপ-টু-ডেট ডকুমেন্টেশনের জন্য, দেখুন "
 
 [deprecation_file_warning]
 other = "ডেপ্রিক্যাটেড"
 
+[outdated_content_title]
+other = "এই  ডকুমেন্টে তথ্য পুরানো হতে পারে"
+
+[outdated_content_message]
+other = "এই  ডকুমেন্টটির আসলটির চেয়ে পুরানো আপডেটের তারিখ রয়েছে, তাই এতে থাকা তথ্য পুরানো হতে পারে৷ আপনি ইংরেজি পড়তে সক্ষম হলে, সবচেয়ে আপ-টু-ডেট তথ্যের জন্য ইংরেজি সংস্করণটি দেখুন:"
```
Comment (sajjadrahman56) [inline data/i18n/bn/bn.toml:102]: এই ডকুমেন্টটির  
remove extra space these 2 words

File: data/i18n/bn/bn.toml [hunk 3]
```diff
@@ -325,25 +425,100 @@ other = "এই নিবন্ধটি এক বছরেরও বেশি
...
+# Please do not localize or modify <current-version> and <current-changelog-url>  placeholders.
+[release_binary_section_note]
+other = """To download older patch versions of <current-version> Kubernetes components (and their checksums), 
+please refer to the [CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG/CHANGELOG-<current-changelog-url>.md) file.""" 
+
+[release_binary_version]
+other  = "সংস্করণ"
+
+[release_binary_version_option]
+other = "সর্বশেষ সংস্করণ"
```
Comment (sajjadrahman56) [inline data/i18n/bn/bn.toml:506]: সর্বশেষ ভার্সন

File: data/i18n/bn/bn.toml [hunk 4]
```diff
@@ -395,26 +570,29 @@ other = "সাবস্ক্রাইব"
...
+[thirdparty_message_vendor]
+other = """এই পৃষ্ঠার আইটেমগুলি কুবারনেটিসের বাইরের বিক্রেতাদের উল্লেখ করে। কুবারনেটিস প্রকল্পের লেখকরা সেই তৃতীয় পক্ষের পণ্য বা প্রকল্পগুলির জন্য দায়ী নয়৷ এই তালিকায় একটি বিক্রেতা, পণ্য বা প্রকল্প যোগ করতে, একটি পরিবর্তন জমা দেওয়ার আগে <a href="/docs/contribute/style/content-guide/#third-party-content">কন্টেন্ট গাইড</a> পড়ুন। <a href="#third-party-content-disclaimer">আরো তথ্য।</a>"""
+
 
 [ui_search_placeholder]
 other = "অনুসন্ধান করুন"
 
 [version_check_mustbe]
-other = "আপনার Kubernetes সার্ভার সংস্করণ হতে হবে "
+other = "আপনার কুবারনেটিস সার্ভার সংস্করণ হতে হবে "
```
Comment (sajjadrahman56) [inline data/i18n/bn/bn.toml:592]: সংস্করণ -> ভার্সন

File: data/i18n/bn/bn.toml [hunk 5]
```diff
@@ -395,26 +570,29 @@ other = "সাবস্ক্রাইব"
...
 
 [version_check_mustbeorlater]
-other = "আপনার Kubernetes সার্ভার সংস্করণের চেয়ে বা তার পরে হতে হবে "
+other = "আপনার কুবারনেটিস সার্ভার সংস্করণের চেয়ে বা তার পরে হতে হবে "
 
 [version_check_tocheck]
 other = "সংস্করণ পরীক্ষা করতে, এন্টার করুন "
```
Comment (sajjadrahman56) [inline data/i18n/bn/bn.toml:598]: সংস্করণ => ভার্সন

**Review by sajjadrahman56 — COMMENTED** [review]
@asem-hamid you can fix now

File: data/i18n/bn/bn.toml [hunk 6]
```diff
@@ -126,11 +201,25 @@ other = "চূড়ান্ত প্যাচ রিলিজ"
...
 other = "এই বিষয়বস্তু দেখতে JavaScript [চালু](https://www.enable-javascript.com/) থাকতে হবে"
 
+[katacoda_message]
+other = """<h4>ইন্টারেক্টিভ টিউটোরিয়াল বন্ধ করুন</h4>
+<p>এই ওয়েবসাইটের ইন্টারেক্টিভ টিউটোরিয়াল বন্ধ করা হচ্ছে। 
+কুবারনেটিস প্রজেক্ট দীর্ঘমেয়াদে একই ধরনের ইন্টারেক্টিভ লার্নিং 
+অপশন পুনঃস্থাপন করার আশা করছে।</p>
+<p>O'Reilly Media's 2019 Katacoda এর<a
+href="https://www.oreilly.com/content/oreilly-acquires-katacoda-and-a-new-way-for-2-5m-customers-to-learn/">অধিগ্রহণ</a>
+অনুসরণ করে শাটডাউন করা হলো ।</p>
```
Comment (sajibAdhi) [inline data/i18n/bn/bn.toml:216]:
```
<p>শাটডাউনটি O'Reilly Media  এর 2019  সালে Katacoda এর <a
href="https://www.oreilly.com/content/oreilly-acquires-katacoda-and-a-new-way-for-2-5m-customers-to-learn/">অধিগ্রহণের</a>
পরে।</p>
```

File: data/i18n/bn/bn.toml [hunk 7]
```diff
@@ -126,11 +201,25 @@ other = "চূড়ান্ত প্যাচ রিলিজ"
...
+<p>Kubernetes o'Reilly এবং Katacoda-এর কাছে অনেক বছর ধরে কুবারনেটইস শেখার জন্য 
+তাদের প্রথম পদক্ষেপ নিতে সাহায্য করার জন্য কৃতজ্ঞ।</p>
```
Comment (sajibAdhi) [inline data/i18n/bn/bn.toml:218]:
```
<p>কুবারনেটিস O'Reilly এবং Katacoda-এর কাছে কৃতজ্ঞ, অনেক বছর ধরে মানুষকে কুবারনেটিস শেখানোর জন্য 
তাদের প্রথম পদক্ষেপ নিতে সাহায্য করার জন্য।</p>
```

File: data/i18n/bn/bn.toml [hunk 8]
```diff
@@ -126,11 +201,25 @@ other = "চূড়ান্ত প্যাচ রিলিজ"
...
+<p>টিউটোরিয়ালগুলি মার্চ, 2023</b> এর <b>31<sup>st</sup> পরে কাজ করা বন্ধ হয়ে 
+যাবে।</b> আরও তথ্যের জন্য, দেখুন "<a href="/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">Free Katacoda Kubernetes Tutorials Are Shutting
+Down</a>."</p>"""
```
Comment (sajibAdhi) [inline data/i18n/bn/bn.toml:221]:
```
<p>টিউটোরিয়ালগুলি <b>৩১শে মার্চ, ২০২৩</b> এর পরে কাজ করা বন্ধ করে
দেবে। আরও তথ্যের জন্য,  "<a href="/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">Free Katacoda Kubernetes Tutorials Are Shutting
Down</a>" দেখুন।</p>"""
```

File: data/i18n/bn/bn.toml [hunk 9]
```diff
@@ -252,22 +341,30 @@ other = "অথবা"
...
 [layouts_docs_search_fetching]
 other = "ফলাফল আনা হচ্ছে..."
 
+[legacy_repos_message]
+other = """The legacy package repositories (`apt.kubernetes.io` and `yum.kubernetes.io`) have been
+[deprecated and frozen starting from September 13, 2023](/blog/2023/08/31/legacy-package-repository-deprecation/).
+**Using the [new package repositories hosted at `pkgs.k8s.io`](/blog/2023/08/15/pkgs-k8s-io-introduction/)
+is strongly recommended and required in order to install Kubernetes versions released after September 13, 2023.**
+The deprecated legacy repositories, and their contents, might be removed at any time in the future and without
+a further notice period. The new package repositories provide downloads for Kubernetes versions starting with v1.24.0."""
```
Comment (sajibAdhi) [inline data/i18n/bn/bn.toml:355]:
```
other = """লিগেসি প্যাকেজ রিপোজিটরিজগুলি (`apt.kubernetes.io` এবং `yum.kubernetes.io`)
[১৩ সেপ্টেম্বর, 2023 থেকে বন্ধ করা হয়েছে এবং ফ্রযেন করা হয়েছে](/blog/2023/08/31/legacy-package-repository-deprecation/)।
**[`pkgs.k8s.io`-এ হোস্ট করা নতুন প্যাকেজ রিপোজিটরিজগুলি](/blog/2023/08/15/pkgs-k8s-io-introduction/) ব্যবহার
করা দৃঢ়ভাবে সুপারিশ করা হয় এবং ১৩ সেপ্টেম্বর, ২০২৩-এর পরে প্রকাশিত কুবারনেটিস ভার্সনগুলি ইনস্টল করার জন্য প্রয়োজনীয়।**
অপ্রচলিত লিগেসি  রিপোজিটরিজগুলি, এবং তাদের কনটেন্ট, ভবিষ্যতে যেকোনো সময় সরানো হতে পারে এবং পরবর্তী
নোটিশের সময় ছাড়াই। নতুন প্যাকেজ রিপোজিটরিজগুলি v1.24.0 থেকে শুরু করে কুবারনেটিস ভার্সনগুলির জন্য ডাউনলোড প্রদান করে।"""
```

Comment (sajibAdhi) [reply data/i18n/bn/bn.toml:355]: if the blogs are localized then add `(/bn/blog/2023/08/31/legacy-package-repository-deprecation/)।`

**Review by sajibAdhi — CHANGES_REQUESTED** [review]
Hey @asem-hamid, you kindly change those are requested.

File: data/i18n/bn/bn.toml [hunk 10]
```diff
@@ -1,26 +1,50 @@
...
 [caution]
 other = "সতর্কতা:"
 
 [cleanup_heading]
 other = "ক্লিনিং"
 
+# Localization note (no need to translate this note)
+# This is always followed by Chinese text, as required by the Kubernetes
+# project's ICP license, issued by the government of the People's Republic
+# of China.
```
Comment (sftim) [inline data/i18n/bn/bn.toml:33]:
```suggestion
```

File: data/i18n/bn/bn.toml [hunk 11]
```diff
@@ -1,26 +1,50 @@
...
+[china_icp_license]
+other = "ICP license:"
```
Comment (sftim) [inline data/i18n/bn/bn.toml:35]:
```suggestion
other = "ICP লাইসেন্স"
```
?

File: data/i18n/bn/bn.toml (hunk 1 above)
Comment (sftim) [reply data/i18n/bn/bn.toml:53]: Wikipedia suggests স্ট্যাক ওভারফ্লো

File: data/i18n/bn/bn.toml [hunk 12]
```diff
@@ -29,29 +53,32 @@ other = "Slack"
...
 [cve_summary]
-other = "Issue Summary"
+other = "ইস্যু সারাংশ"
 
 [cve_table]
-other = "Official Kubernetes CVE List"
+other = "অফিসিয়াল কুবারনেটিস CVE তালিকা"
 
 [cve_table_date_format]
 other = "02 Jan 2006 15:04:05 MST"
```
Comment (sftim) [inline data/i18n/bn/bn.toml:84]: Can change this format, BTW. How about
```suggestion
other = "2006-01-02 15:04:05 MST"
```

File: data/i18n/bn/bn.toml [hunk 13]
```diff
@@ -96,19 +132,58 @@ other = "পুরানো সংস্করণ"
...
+other = "ডেপ্রিক্যাটেড"
+
+[feature_gate_table_header_default]
+other = "ডিফল্ট"
+
+[feature_gate_table_header_feature]
+other = "ফিচার"
+
+[feature_gate_table_header_from]
+other = "From"
```
Comment (sftim) [inline data/i18n/bn/bn.toml:162]: I'd expect Bangla here

File: data/i18n/bn/bn.toml [hunk 14]
```diff
@@ -96,19 +132,58 @@ other = "পুরানো সংস্করণ"
...
+
+[feature_gate_table_header_since]
+other = "Since"
```
Comment (sftim) [inline data/i18n/bn/bn.toml:165]: I'd expect Bangla here

File: data/i18n/bn/bn.toml [hunk 15]
```diff
@@ -96,19 +132,58 @@ other = "পুরানো সংস্করণ"
...
+
+[feature_gate_table_header_stage]
+other = "Stage"
+
+[feature_gate_table_header_to]
+other = "To"
```
Comment (sftim) [inline data/i18n/bn/bn.toml:171]: I'd expect Bangla here

File: data/i18n/bn/bn.toml [hunk 16]
```diff
@@ -96,19 +132,58 @@ other = "পুরানো সংস্করণ"
...
+
+[feature_gate_table_header_until]
+other = "Until"
```
Comment (sftim) [inline data/i18n/bn/bn.toml:174]: I'd expect Bangla here

File: data/i18n/bn/bn.toml (hunk 9 above)
Comment (sftim) [reply data/i18n/bn/bn.toml:355]: https://github.com/kubernetes/website/pull/46271/files#r1594992002 as a committable suggestion:
```suggestion
other = """লিগেসি প্যাকেজ রিপোজিটরিজগুলি (`apt.kubernetes.io` এবং `yum.kubernetes.io`)
[১৩ সেপ্টেম্বর, 2023 থেকে বন্ধ করা হয়েছে এবং ফ্রযেন করা হয়েছে](/blog/2023/08/31/legacy-package-repository-deprecation/)।
**[`pkgs.k8s.io`-এ হোস্ট করা নতুন প্যাকেজ রিপোজিটরিজগুলি](/blog/2023/08/15/pkgs-k8s-io-introduction/) ব্যবহার
করা দৃঢ়ভাবে সুপারিশ করা হয় এবং ১৩ সেপ্টেম্বর, ২০২৩-এর পরে প্রকাশিত কুবারনেটিস ভার্সনগুলি ইনস্টল করার জন্য প্রয়োজনীয়।**
অপ্রচলিত লিগেসি  রিপোজিটরিজগুলি, এবং তাদের কনটেন্ট, ভবিষ্যতে যেকোনো সময় সরানো হতে পারে এবং পরবর্তী
নোটিশের সময় ছাড়াই। নতুন প্যাকেজ রিপোজিটরিজগুলি v1.24.0 থেকে শুরু করে কুবারনেটিস ভার্সনগুলির জন্য ডাউনলোড প্রদান করে।"""
```

File: data/i18n/bn/bn.toml [hunk 17]
```diff
@@ -325,25 +425,100 @@ other = "এই নিবন্ধটি এক বছরেরও বেশি
...
 [prerequisites_heading]
 other = "শুরু করার আগে"
 
 [previous_patches]
 other = "প্যাচ রিলিজ:"
 
+# The following text is displayed when JavaScript isn't available.
+[release_binary_alternate_links]
+other  = """You can find links to download Kubernetes components (and their checksums) in the [CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG) files.
+Alternately, use [downloadkubernetes.com](https://www.downloadkubernetes.com/) to filter by version and architecture."""
```
Comment (sftim) [inline data/i18n/bn/bn.toml:439]: I'd expect to see Bangla here

File: data/i18n/bn/bn.toml [hunk 18]
```diff
@@ -325,25 +425,100 @@ other = "এই নিবন্ধটি এক বছরেরও বেশি
...
+other = "আর্কিটেকচারগুলো"
+
+[release_binary_copy_link]
+other = "কপি লিংক"
+
+[release_binary_copy_link_certifcate]
+other = "cert"
+
+[release_binary_copy_link_checksum]
+other = "checksum"
```
Comment (sftim) [inline data/i18n/bn/bn.toml:454]: Is there a word for “checksum” in Bangla?

File: data/i18n/bn/bn.toml [hunk 19]
```diff
@@ -325,25 +425,100 @@ other = "এই নিবন্ধটি এক বছরেরও বেশি
...
+
+[release_binary_table_caption]
+other = "কুবারনেটিস কম্পোনেন্ট বাইনারি ডাউনলোড করুন"
+
+# NOTE: <current-version> is a placeholder for the actual version number set by 'release-binaries' shortcode.
+# Please do not localize or modify <current-version> placeholder.
+[release_binary_section]
+other = """You can find the links to download <current-version> Kubernetes components (along with their checksums) below.
+To access downloads for older supported versions, visit the respective documentation 
+link for [older versions](https://kubernetes.io/docs/home/supported-doc-versions/#versions-older) or use [downloadkubernetes.com](https://www.downloadkubernetes.com/)."""
```
Comment (sftim) [inline data/i18n/bn/bn.toml:494]: I'd expect to see Bangla here

File: data/i18n/bn/bn.toml [hunk 20]
```diff
@@ -325,25 +425,100 @@ other = "এই নিবন্ধটি এক বছরেরও বেশি
...
+[release_binary_section]
+other = """You can find the links to download <current-version> Kubernetes components (along with their checksums) below.
+To access downloads for older supported versions, visit the respective documentation 
+link for [older versions](https://kubernetes.io/docs/home/supported-doc-versions/#versions-older) or use [downloadkubernetes.com](https://www.downloadkubernetes.com/).""" 
+
+# NOTE: <current-version> and <current-changelog-url> are placeholders set by 'release-binaries' shortcode.
+# Please do not localize or modify <current-version> and <current-changelog-url>  placeholders.
+[release_binary_section_note]
+other = """To download older patch versions of <current-version> Kubernetes components (and their checksums), 
+please refer to the [CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG/CHANGELOG-<current-changelog-url>.md) file."""
```
Comment (sftim) [inline data/i18n/bn/bn.toml:500]: I'd expect to see Bangla here. You can omit this if you don't want to add a translation yet.

File: data/i18n/bn/bn.toml [hunk 21]
```diff
@@ -325,25 +425,100 @@ other = "এই নিবন্ধটি এক বছরেরও বেশি
...
-other = "(রিলীস হয়েছিল: "
+other = "(রিলিজ হয়েছিল: "
 
 # See https://gohugo.io/functions/format/gos-layout-string
 # Use a suitable format for your locale
 [release_date_format]
 other = "2006-01-02"
 
+[release_date_format_month]
+other = "January 2006"
```
Comment (sftim) [inline data/i18n/bn/bn.toml:520]: I'd use a format that doesn't require people to speak English.

**Review by sftim — COMMENTED** [review]
For any entry, if you want to use the English text, all you do is leave an entry out of the Bangla file. It should be very rare to see entries in this file that are in all-Roman letters.

File: data/i18n/bn/bn.toml (hunk 10 above)
Comment (asem-hamid) [reply data/i18n/bn/bn.toml:33]: Can we please keep this? It will help us maintain a certain line break, which is further helpful for updating this file.

Comment (sftim) [reply data/i18n/bn/bn.toml:33]: Sure, it's fine to keep it.

File: data/i18n/bn/bn.toml (hunk 1 above)
Comment (asem-hamid) [reply data/i18n/bn/bn.toml:53]: স্ট্যাক ওভারফ্লো is good

File: data/i18n/bn/bn.toml (hunk 12 above)
Comment (asem-hamid) [reply data/i18n/bn/bn.toml:84]: it's depend on  en team , 

In Bangladesh we use 02 Jan 2006  or 02-02-2006  ,
Also most of the people here aware of that people around the world use  YYYY-MM-DD or MM-DD-YYYY

what's your suggestion

File: data/i18n/bn/bn.toml [hunk 22]
```diff
@@ -96,19 +132,58 @@ other = "পুরানো সংস্করণ"
 other = "জীবনের শেষ:"
```
[the remark this answers is not in the document]
Comment (asem-hamid) [reply data/i18n/bn/bn.toml:132]: going with 

[docs_version_latest_heading]
other = "সর্বশেষ ভার্সন"

[docs_version_other_heading]
other = "পুরানো ভার্সন"

File: data/i18n/bn/bn.toml (hunk 9 above)
Comment (asem-hamid) [reply data/i18n/bn/bn.toml]: can one string have 2 other ?

image

File: data/i18n/bn/bn.toml (hunk 18 above)
Comment (asem-hamid) [reply data/i18n/bn/bn.toml:454]: There is no actual word but we can use Banlish (english word but typed in bangla ) 

"চেকসাম"

File: data/i18n/bn/bn.toml (hunk 12 above)
Comment (asem-hamid) [reply data/i18n/bn/bn.toml:84]: oh I [understand](https://github.com/kubernetes/website/pull/46271/files#r1595126057) ,,

File: data/i18n/bn/bn.toml (hunk 21 above)
Comment (asem-hamid) [reply data/i18n/bn/bn.toml:520]: yeah we can use localize version of month name ,,,

**Comment by asem-hamid** [comment]
@sajjadrahman56 , @sajibAdhi , @sftim  , @mitul3737 

Done

**Review by sajibAdhi — COMMENTED** [review]
/lgtm

**Pull request merged**